testfiles/PlantLoopHeatPump_EIR_AirSource_and_AWHP_cooling_only.idf

```diff
@@ -0,0 +1,497 @@
+    0.0,                     !- Coefficient2 x
+    0.0,                     !- Coefficient3 x**2
+    0.0,                     !- Minimum Value of x
+    1.0;                     !- Maximum Value of x
+
+  BranchList,
+    Cooling Coil Load Loop Demand Branches,  !- Name
+    Cooling Coil Load Loop Demand Branch;  !- Branch 1 Name
+
+  Branch,
+    Cooling Coil Load Loop Demand Branch,  !- Name
+    ,                        !- Pressure Drop Curve Name
+    LoadProfile:Plant,       !- Component 1 Object Type
+    Cooling Coil Load Profile,  !- Component 1 Name
+    Cooling Coil Load Loop Demand Inlet Node,  !- Component 1 Inlet Node Name
+    Cooling Coil Load Loop Demand Outlet Node;  !- Component 1 Outlet Node Name
+
+  LoadProfile:Plant,
+    Cooling Coil Load Profile,  !- Name
+    Cooling Coil Load Loop Demand Inlet Node,  !- Inlet Node Name
+    Cooling Coil Load Loop Demand Outlet Node,  !- Outlet Node Name
+    Cooling Coil Load Profile Load Schedule,  !- Load Schedule Name
+    0.005,                   !- Peak Flow Rate {m3/s}
+    Cooling Coil Load Profile Flow Frac Schedule;  !- Flow Rate Fraction Schedule Name
+
+  OutdoorAir:Node,
+    Outdoor Air Inlet Node;  !- Name
+
+  Output:VariableDictionary,IDF;
+
+  Output:Variable,*,Heat Pump Operating Mode is Cooling,detailed;  !- HVAC Average []
+  Output:Variable,*,Operating Model Output Cooling,detailed;
+  Output:Variable,*,Outdoor Temp,detailed;
+
+  OutputControl:Table:Style,
+    HTML;                    !- Column Separator
+
+  Output:Table:SummaryReports,
+    AllSummary;              !- Report 1 Name
+
+  Output:EnergyManagementSystem,
+    Verbose,                 !- Actuator Availability Dictionary Reporting
+    Verbose,                 !- Internal Variable Availability Dictionary Reporting
+    Verbose;                 !- EMS Runtime Language Debug Output Level
+
+  EnergyManagementSystem:Actuator,
+    opModeAct_cooling,       !- Name
+    test_AWHP,               !- Actuated Component Unique Name
+    HeatPump:AirToWater:Cooling,  !- Actuated Component Type
+    Operating Mode;          !- Actuated Component Control Type
+
+  EnergyManagementSystem:OutputVariable,
+    Operating Model Output Cooling,  !- Name
+    opModeAct_cooling,       !- EMS Variable Name
+    Averaged,                !- Type of Data in Variable
+    SystemTimeStep,          !- Update Frequency
+    ,                        !- EMS Program or Subroutine Name
+    ;                        !- Units
+
+  EnergyManagementSystem:ProgramCallingManager,
+    Change_OpMode_EMS_Program_Manager,  !- Name
+    InsideHVACSystemIterationLoop,  !- EnergyPlus Model Calling Point
+    Change_OpMode_EMS_Program;  !- Program Name 1
+
+  EnergyManagementSystem:Program,
+    Change_OpMode_EMS_Program,  !- Name
+    IF (Hour <= 12),         !- Program Line 2
+    SET opModeAct_cooling = 0,  !- <none>
+    ELSE,                    !- <none>
+    SET opModeAct_cooling = 1,  !- <none>
+    ENDIF;                   !- <none>
```

Comment:
I don't get this EMS program here?

Again, this ends up being an Example File.